src/main/services/uml-element/resizable/resizing-reducer.ts

```diff
@@ -1,8 +1,37 @@
 import { Reducer } from 'redux';
+import { IBoundary } from '../../../utils/geometry/boundary';
 import { Actions } from '../../actions';
 import { ResizingActionTypes, ResizingState } from './resizing-types';
 
 export const ResizingReducer: Reducer<ResizingState, Actions> = (state = {}, action) => {
+  const obj = { x: 0, y: 0 };
+
+  const getUpdatedPosition = (elem: IBoundary, payload: any, resizeFrom: string) => {
```

Comment:
we need to restrict the new positions to not be smaller or greater than the opposing corners via Math.max or Math.min.

Sent you the specifics.